src/runtime/internal/dom.js

```diff
@@ -993,7 +993,7 @@ export { ResizeObserverSingleton };
 /**
  * @returns {void} */
 export function toggle_class(element, name, toggle) {
-	element.classList[toggle ? 'add' : 'remove'](name);
+	element.classList.toggle(name, !!toggle);
```

Comment:
the coercion isn't necessary, the `toggle` method doesn't care

```suggestion
	element.classList.toggle(name, toggle);
```